lib/tasks/hourglass.rake

```diff
@@ -44,15 +44,17 @@ namespace :hourglass do
   task :monthly_report, [:date] => :environment do |_, args|
     Rails.logger.silence do
       date = args[:date] ? Date.parse(args[:date]) : Date.current.last_month
-      puts GenerateReport.call!(range: date.all_month).output
+      tags = (ENV["TAGS"] || ENV["TAG"]).to_s.split(",")
```

Comment:
These tasks are now using both rake variables and environment variables. Should we choose one or the other here to stay consistent?